code/modules/cargo/packs.dm

```diff
@@ -2032,6 +2032,15 @@
 	crate_name = "Bicycle Crate"
 	crate_type = /obj/structure/closet/crate/large
 
+/datum/supply_pack/misc/excercise_bikes
+	name = "Excercise Bikes"
+	desc = "For when the engineers delaminate the SM and all your pacmen are out of fuel, this is a 5-pack! (Remember to anchor them into a power source before using!)"
```

Comment:
`For when the engineers delaminate the SM and all your PACMANs are out of fuel. This is a 5-pack!`

PACMAN is an acronym.

Reply:
What does PACMAN stand for? PACMANs might be a double plural.